src/types/chart/common_bar_combo.ts

```diff
@@ -6,6 +6,8 @@ export interface ComboBarChartDefinition {
   readonly dataSetsHaveTitle: boolean;
   readonly labelRange?: string;
   readonly title: string;
+  readonly isTitleCellReferenced?: boolean;
```

Comment:
nitpick: I think i'd rather have something like `titleType: "reference" | "rawValue"`

Reply:
Hello, Firstly, I appreciate your input.

If we maintain

```js
readonly title: Range | string;
readonly titleType: "reference" | "value";
```

Then we have to include the title in the runtime of each chart. right? Otherwise, it will raise an error when retrieving the default Chart.js runtime because there is a chance that the title may contain Range. So when passing data we also need to pass title with it. Please correct me if i am wrong

And in case of excel export, while getting definition, if title type is reference then we need to set the title manually by getting getCellText, but is not possible with core getters right?

Reply:
- The range should be converted to the cell's value when creating the runtime
- For excel export, we can define `exportForExcel` in an UI plugin to overwrite the cell reference to its value